recipes/geocat-datafiles/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "geocat-datafiles" %}
+{% set version = "2022.03.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/{{ name }}/archive/v{{ version }}.tar.gz # For building from a specific tag's tarball
+  sha256: 68aef597dbe4223fff37b11dab5e8720b1047085af58c95ddfa27d86c5fb75c9
+  # git_tag: v{{ version }} # For building from a specific tag
+  # git_rev: main # For building from branch for debugging purposes
+  # git_url: https://github.com/NCAR/{{ name }}.git
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed -vv .
+  # skip: true  # [py<38 or win]
+
+requirements:
+  build:
+    - python >=3.8                        # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools                          # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - pooch                               # [build_platform != target_platform]
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+    - pooch
+  run:
+    - python >=3.8
+    - pooch
+
+test:
+  imports:
+    - geocat.datafiles
+
+about:
+  home: https://geocat.ucar.edu/
+  summary: 'GeoCAT-datafiles is a Python module that allows users to download sample datasets for use with GeoCAT projects.'
+  dev_url: https://github.com/NCAR/{{ name }}
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - pilotchute # A. Kootz https://github.com/pilotchute
+    - erogluorhan # O. Eroglu https://github.com/erogluorhan
+    - anissa111 # A. Zacharias https://github.com/anissa111
+    - michaelavs # M. Sizemore https://github.com/michaelavs
```

Comment:
Everyone listed should consent in a comment here ideally, unless this is like a common team where everyone goes on projects together :)

(Alternatively, you can just add yourself for now, and later in a feedstock, you can add the others)

Reply:
Yes, but...

> * [x]  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

from the main body of this PR (you incorrectly checked this item off)

Reply:
We don't want to be spamming people with notifications they don't want, so every maintainer for every recipe has to comment that they're willing to be listed as a maintainer (or add a :+1: reaction).

Reply:
I am willing to be a maintainer

Reply:
All comments are made.